3S/jsonld-contexts/3S.jsonld

```diff
@@ -0,0 +1,52 @@
+{
+	"@context": {
+		"Client": "https://ontology.eglobalmark.com/3S#fishNumber",
+		"ServicePoint": "https://ontology.eglobalmark.com/3S#ServicePoint",
+		"country": "https://ontology.eglobalmark.com/3S#country",
+		"timeZone": "https://ontology.eglobalmark.com/3S#timeZone",
+		"hydroSector": "https://ontology.eglobalmark.com/3S#hydroSector",
+		"refClient": "https://ontology.eglobalmark.com/3S#refClient",
+		"refServicePoint": "https://ontology.eglobalmark.com/3S#refServicePoint",
+		"refTransmitter": "https://ontology.eglobalmark.com/3S#refTransmitter",
+		"refWatertMeter": "https://ontology.eglobalmark.com/3S#refWatertMeter",
+		"sensorType": "https://ontology.eglobalmark.com/3S#sensorType",
+		"diameter": "https://ontology.eglobalmark.com/3S#diameter",
+		"pulseWeight": "https://ontology.eglobalmark.com/3S#pulseWeight",
+		"dateRemoved": "https://ontology.eglobalmark.com/3S#dateRemoved",
+		"duration": "https://ontology.eglobalmark.com/3S#duration",
+
+		"index_15m_l": "https://ontology.eglobalmark.com/3S#index_15m_l",
```

Comment:
are you sure that your properties names are compatible with what the spec is requesting?